.github/workflows/gradle-build.yml

```
@@ -21,8 +21,8 @@ jobs:
       - name: Set up JDK
         uses: actions/setup-java@v4
         with:
-          java-version: 18
-          distribution: 'adopt'
+          java-version: 20
```

```suggestion
          java-version: 21
```
21 is an Lts, 20 isn't